scripts/ci_build_rust.bash

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -ex
+
+VERSION="$(git describe --always --tags --dirty)"
+
+if [ "$RUNNER_OS" == "Linux" ] || [ "$RUNNER_OS" == "macOS" ]; then
+    BUILD_TRIPLET="$(cc -dumpmachine)"
+    ARTIFACT_NAME="sbp_tools-${VERSION}-${BUILD_TRIPLET}.zip"
+    EXECUTABLES=("sbp2json" "json2sbp" "json2json")
+    PACKAGE_CMD="zip ../../$ARTIFACT_NAME "${EXECUTABLES[@]}""
```

Comment:
fwiw,..
```
❯ shellcheck scripts/ci_build_rust.bash

In scripts/ci_build_rust.bash line 11:
    PACKAGE_CMD="zip ../../$ARTIFACT_NAME "${EXECUTABLES[@]}""
                ^-- SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
                                           ^---------------^ SC2027: The surrounding quotes actually unquote this. Remove or escape them.


In scripts/ci_build_rust.bash line 16:
    PACKAGE_CMD="7z a -tzip ../../$ARTIFACT_NAME "${EXECUTABLES[@]}""
                ^-- SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
                                                  ^---------------^ SC2027: The surrounding quotes actually unquote this. Remove or escape them.

For more information:
  https://www.shellcheck.net/wiki/SC2027 -- The surrounding quotes actually u...
  https://www.shellcheck.net/wiki/SC2124 -- Assigning an array to a string! A...
```